pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>sinonjs</artifactId>
-    <version>1.7.3-2-SNAPSHOT</version>
+    <version>1.17.2-1-SNAPSHOT</version>
```

Comment:
No problem, sorted.

Date: Sun, 24 Jan 2016 16:13:04 -0800
From: notifications@github.com
To: sinonjs@noreply.github.com
CC: fmcgough@live.co.uk
Subject: Re: [sinonjs] Update sinon.js version to 1.17.2 (#2)

In pom.xml:

> @@ -11,7 +11,7 @@
>      <packaging>jar</packaging>
>      <groupId>org.webjars</groupId>
>      <artifactId>sinonjs</artifactId>
> -    <version>1.7.3-2-SNAPSHOT</version>
> +    <version>1.17.2-1-SNAPSHOT</version>

Usually we only do the -1 thing for a revision / fix to the same upstream version.  So would you mind changing this to 1.17.2-SNAPSHOT ?


—
Reply to this email directly or view it on GitHub.